README.md

```diff
@@ -52,35 +52,50 @@ Then, we unzip it to the folder where we want Maven to live.
 
 2. You'll see skill-tree-backed (If the process is running)
 
-3. skill-tree-backend>skill-tree-backend-db-1>open in terminal
+3. skill-tree-backend > skill-tree-backend-db-1 > open in terminal
  
-to login to MySQL
-mysql -u root -p (in terminal)
-password : rootpassword
+To login to MySQL
+```
+mysql -u root -p
+```
+(in terminal)
+password : password
+
+Reference Screenshots:
+
+If the project is started with ```docker compose up```, this can be seen once you open Docker Desktop:
 
-Refrence Screenshots:
-If the project is started with docker compose up thise can be seen once you open Docker Desktop:
 <img width="1680" alt="Screenshot 2023-12-26 at 9 33 17 PM" src="https://github.com/ashifkhn/skill-tree-backend/assets/54736284/57b90473-ae22-45b2-8a19-3377bfbcf1b9">
 
 
+
 Terminal needs to be opened here:
+
 ![image](https://github.com/ashifkhn/skill-tree-backend/assets/54736284/d66166ae-b931-40ab-914f-f42615323a32)
 
 
 
-## Steps for Creating the Database
+## Steps for Creating the Database (To be executed in order)
+
+Semicolon is important in the commands
+
+Username: testuser
+Password: testpassword
 
-1. `create database skilltree;`(semicolon is important here)
-2. `show databases;`
-3. `create user 'testuser' identified by 'testpassword';` (Username: testuser, Password: testpassword)
-4. `grant all on skilltree.* to testuser;`
+```
+CREATE DATABASE skilltree;
+SHOW DATABASES;
+CREATE USER 'testuser' IDENTIFIED BY 'testpassword';
+GRANT ALL ON skilltree.* TO testuser;
+```
 
-## Steps for Creating the Database
+## Steps for setting up skill-tree in Intellij
 
 1. After creating the database project needs to be compiled.
 2. Open skill-tree-backend in intellij.
 3. Java_Home path needs to be added here.
-4. You can either add the existing path and jdk 17 can be downloaded inside intellij.
+4. You can either add the existing path and JDK 17 can be downloaded inside intellij.
+5. If adding from the existing path, go to Settings > Project Structure > Choose the earlier installed Java 17 SDK.
```

Comment:
Missing steps here, I can't see how will the other person load env file, also this points can be more clear

Reply:
I have updated with a new section having steps to create .env file and run configurations